### PR TITLE
Buttons

### PR DESCRIPTION
**Pull request by rishabh486**
pr created for button component
Screenshot (30)
t

### CODE REVIEW

File: button.css [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+}
+.button-container {
+  padding: 1rem 2rem;
+  margin: 1rem;
+  /* margin-left:-0.3rem; */
+  display: inline-block;
+  font-weight: 800;
+  font-size: 1.8rem;
+  border: none;
+  border-radius: 1rem 1rem 1rem 1rem;
```
Comment (saurabhan) [inline button.css]: you can just put border-radius: 1rem it will apply to all corners

File: button.css [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+  color: black;
+  font-family: "roboto";
+}
+.primary-button {
+  background-color: #49a0eb;
```
Comment (saurabhan) [inline button.css]: use var for color scheme, it will be easier to modify and reuse

**Review by saurabhan — APPROVED** [review]
good to merge, do look at suggested changes.

File: button.css (hunk 2 above)
Comment (DivyanshSareen) [reply button.css]: +1

File: button.css [hunk 3]
```diff
@@ -0,0 +1,88 @@
+:root {
+  font-size: 62.5%;
+  box-sizing: border-box;
+}
+.button-container {
+  padding: 1rem 2rem;
+  margin: 1rem;
+  /* margin-left:-0.3rem; */
```
Comment (DivyanshSareen) [inline button.css]: remove code comment

**Review by DivyanshSareen — APPROVED** [review]
Good to merge

File: button.css (hunk 3 above)
Comment (rishabh486) [reply button.css]: i have done the changes

File: button.css (hunk 2 above)
Comment (rishabh486) [reply button.css]: made changes and used color schemes

File: button.css (hunk 1 above)
Comment (rishabh486) [reply button.css]: done with the changes u suggested

**Pull request merged**